From c674b2152e5530fd4601d32f2cb5f455e7ef4809 Mon Sep 17 00:00:00 2001
From: Adam Jackson <ajax@redhat.com>
Date: Wed, 17 Oct 2018 11:44:37 -0400
Subject: [PATCH] Suppress high-keycode warnings at the default warning level

We expect evdev to have keycodes above 255, there's no reason to warn
about this.

Signed-off-by: Adam Jackson <ajax@redhat.com>
---
 keycodes.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/keycodes.c b/keycodes.c
index c21f7c4..425ba10 100644
--- a/keycodes.c
+++ b/keycodes.c
@@ -332,7 +332,7 @@ AddKeyName(KeyNamesInfo * info,
 
     if ((kc < info->effectiveMin) || (kc > info->effectiveMax))
     {
-        if (!high_keycode_warned)
+        if (!high_keycode_warned && warningLevel > 1)
         {
             WARN2("Unsupported high keycode %d for name <%s> ignored\n",
                   kc, name);
@@ -596,7 +596,7 @@ HandleKeycodeDef(KeycodeDef * stmt, unsigned merge, KeyNamesInfo * info)
     code = result.ival;
     if ((code < info->effectiveMin) || (code > info->effectiveMax))
     {
-        if (!high_keycode_warned)
+        if (!high_keycode_warned && warningLevel > 1)
         {
             WARN2("Unsupported high keycode %d for name <%s> ignored\n",
                   code, stmt->name);
-- 
2.19.0

